src/SFA.DAS.Commitments.Infrastructure/Authorization/AuthorizeRemoteOnlyAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Web.Http;
+using System.Web.Http.Controllers;
+
+namespace SFA.DAS.Commitments.Infrastructure.Authorization
+{
+    public class AuthorizeRemoteOnlyAttribute : AuthorizeAttribute
```

Comment:
I know we discussed this, but this may cause us a problem later on when we need to enable AAD authorisation in place of JWT.

Reply:
We'll need to update how we handle authorization in the integration tests when we tackle that change. Ideally, we'd change the integration tests calls to be properly authorized. I attempted that for the current version of the tests, but unfortunately hit what appeared to be a bug in .net, which is why the remote only work-around was introduced. Note, that the remote only attribute was reused from the provider events api, so there is precedent for allowing local requests to be unsecured.